packages/zipkin-encoder-thrift/package.json

```diff
@@ -22,6 +22,6 @@
   "homepage": "https://github.com/openzipkin/zipkin-js#readme",
   "dependencies": {
     "thrift": "^0.11.0",
-    "zipkin": "^0.21.0"
+    "zipkin": "file:../zipkin"
```

Comment:
I see this link https://github.com/lerna/lerna#common-devdependencies specifically talks about `devDependencies` but this change changes `dependencies`, I am curious about what would be the version being pinned for this.

Reply:
the current version in packages/zipkin
currently 0.21.0, but when upgraded 0.22.0

as lerna here works as in fixed(not independent mode) every time we change in one lib, every other upgrades automatically.